packages/hash/api/src/graph/workspace-knowledge.ts

```diff
@@ -0,0 +1,36 @@
+import { Logger } from "@hashintel/hash-backend-utils/logger";
+import { GraphApi } from "@hashintel/hash-graph-client";
+import { NotFoundError } from "../lib/error";
+import { logger } from "../logger";
+import { WorkspaceInstanceModel } from "../model";
+import { workspaceAccountId } from "../model/util";
+
+/**
+ * Ensures the required workspace knowledge has been created in the graph.
+ */
+export const ensureWorkspaceKnowledgeExists = async (params: {
```

Comment:
I've mentioned separately that we use "Workspace" inconsistently in discussions to mean either a namespace to which entities belong (e.g. a user or org's "workspace") and the app as a whole ("the workspace", this usage).

I think it would be less cause for confusion if we replaced this usage of 'workspace' with 'system'.

Reply:
We agreed internally to replace uses of "workspace" with "system", which I have done for the changes introduced by this PR in https://github.com/hashintel/hash/pull/1327/commits/6f0eafcd7ea3643b56e8da99f93b93834ac4f3db.

I've also renamed the "Workspace Instance" entity type to the "HASH Instance" entity type in https://github.com/hashintel/hash/pull/1327/commits/1c843af8ed5849a0a89af279ff80897fd5162285

Reply:
I will address the remaining uses of "workspace" in the backend in [this follow-up task](https://app.asana.com/0/1202805690238892/1203322851277858/f)